massgov/openmass
# DP-22403: Make existing paragraph types on org page available for sections (follow-up)

**Description:**
Follow up PR from internal review.

**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22403

**To Test:**
- Login and edit an Organization.
- Click on the Content tab.
- Add an Organization Section.
- Add a Featured Item Mosaic.
- [ ] Verify you can't add less than 5 items.
- [ ] Verify you can't save without the first item having a Highlight Image.

**Screenshots/GIFs:**

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
